test/api_integration/apis/stats/stats.js

```diff
@@ -15,7 +15,7 @@ const assertStatsAndMetrics = (body) => {
   expect(body.kibana.transport_address).to.be.a('string');
   expect(body.kibana.version).to.be.a('string');
   expect(body.kibana.snapshot).to.be.a('boolean');
-  expect(body.kibana.status).to.be('green');
+  expect(body.kibana.status).to.be.a('string');
```

Comment:
The test is not supposed to assert that kibana is green and just check the expected shape of the payload, so I just changed this assertion to have it not fails when kibana is yellow during the test for some reason